File: app/src/main/java/de/nicidienase/chaosflix/activities/ConferencesActivity.java
package de.nicidienase.chaosflix.activities;

import android.os.Bundle;

import de.nicidienase.chaosflix.R;

/**
 * Created by felix on 18.03.17.
 */

public class ConferencesActivity extends AbstractServiceConnectedAcitivty {
	@Override
	protected void onCreate(Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
//		setContentView(R.layout.activity_conferences_grid);
		setContentView(R.layout.activity_conferences_browse);
	}
}

File: leanback/src/main/java/de/nicidienase/chaosflix/leanback/conferences/ConferencesActivity.kt
package de.nicidienase.chaosflix.leanback.conferences

import android.os.Bundle
import de.nicidienase.chaosflix.leanback.R
import de.nicidienase.chaosflix.leanback.recommendations.RecommendationBroadcastReceiver

class ConferencesActivity : androidx.fragment.app.FragmentActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_conferences_browse)

        RecommendationBroadcastReceiver.setup(this)
    }

    companion object {
        private val TAG = ConferencesActivity::class.java.simpleName
    }
}
